frontend/src/scenes/session-recordings/player/PlayerController.tsx

```diff
@@ -153,6 +163,9 @@ export function PlayerController({
                                     >
                                         Autoplay enabled
                                     </LemonButton>
+                                    <LemonButton status="stealth" fullWidth onClick={() => onAddToPlaylist()}>
+                                        Add recording to static playlist
+                                    </LemonButton>
```

Comment:
One thought here - this might make sense to do as a pre-popup, similar to youtube for example, so you can quickly add to an existing list _or_ create a new playlist which then pops up with the full modal.

Thinking this would be a frustrating flow to have to be jumping in and out of a modal when you're quickly adding recordings to a list

<img width="387" alt="Screenshot 2022-11-17 at 11 17 27" src="https://user-images.githubusercontent.com/2536520/202420047-c76cbf9e-4f13-4394-8637-d733d294fec5.png">

Reply:
Good idea! Right now the interface is copied over from the insight to add dashboard flow so I'll save this cleanup for a future PR